[ERROR] [1603575919.379678]: Error processing request: field env must be of type grasp_path_planner/EnvDescMsg instead of carla_utils/EnvDescMsg
['Traceback (most recent call last):\n', 
'  File "/home/mayank/Fall2020/devel/lib/python2.7/dist-packages/grasp_path_planner/srv/_SimService.py", line 516, in serialize\n 
   buff.write(_get_struct_dBd2BdqB().pack(_x.env.speed_limit, _x.env.reward_info.collision, _x.env.reward_info.time_elapsed, _x.env.reward_info.new_run, _x.env.reward_info.end_of_action, _x.env.reward_info.action_progress, _x.env.reward_info.current_action, _x.env.reward_info.path_planner_terminate))\n'
   , 'struct.error: required argument is not a float\n', '\nDuring handling of the above exception, another exception occurred:\n\n', 
   'Traceback (most recent call last):\n', '  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_service.py", line 629, in _handle_request\n    transport.send_message(response, self.seq)\n', '  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 665, in send_message\n    serialize_message(self.write_buff, seq, msg)\n', '  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msg.py", line 152, in serialize_message\n    msg.serialize(b)\n', '  File "/home/mayank/Fall2020/devel/lib/python2.7/dist-packages/grasp_path_planner/srv/_SimService.py", line 525, in serialize\n    except struct.error as se: self._check_types(struct.error("%s: \'%s\' when writing \'%s\'" % (type(se), str(se), str(locals().get(\'_x\', self)))))\n', '  File "/opt/ros/kinetic/lib/python2.7/dist-packages/genpy/message.py", line 334, in _check_types\n    check_type(n, t, getattr(self, n))\n', '  File "/opt/ros/kinetic/lib/python2.7/dist-packages/genpy/message.py", line 264, in check_type\n    raise SerializationError("field %s must be of type %s instead of %s"%(field_name, field_type, field_val._type))\n', 'genpy.message.SerializationError: field env must be of type grasp_path_planner/EnvDescMsg instead of carla_utils/EnvDescMsg\n']